src/adhocracy_core/thentos.cfg

```diff
@@ -0,0 +1,27 @@
+[buildout]
+parts +=
+    thentos_download
+    thentos_config
+    supervisor
+
+[thentos_download]
+# Download and extract thentos binary tarball
+recipe = hexagonit.recipe.download
+url = https://github.com/liqd/thentos/releases/download/v0.1.0/thentos-captcha-v0.1.0.bin.tar.gz
+strip-top-level-dir = true
+md5sum = 6a863a985a6ff27f319674d214e221ca
+ignore-existing = true
+
+[thentos_config]
+# Configure thentos
+recipe = collective.recipe.cmd
+on_install = true
+on_update = true
+cmds = [ -f ../thentos-captcha.config ] || cp thentos-captcha.config .. 
```

Comment:
we use the inline templates for this, like here: https://github.com/liqd/adhocracy3/blob/master/src/adhocracy_core/checkcode.cfg#L74

..and sorry for the late answer